Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,81 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Node.js Admin Repository
+
+The Node.js Admin Repository (nodejs/admin) will serve the central location for
+managing Node.js GitHub Organization administrative activities. Only Node.js
+GitHub Organization owners will have write permissions to the nodejs/admin
+repository.
+
+## Organization Roles
+
+### Owners
+
+The Node.js Foundation Executive Director, Node.js TSC Director, TSC Chair,
+Community Committee Chair, and Node.js Foundation Education Manager shall be
+the only individuals granted Owner permissions within the Node.js GitHub
+Organization. Should, at any point in the future, the Node.js Foundation
+Board establish a Community Committee Director position equivalent to the
+TSC Director, that individual would also be automatically granted Owner
+permissions within the organization.
+
+Upon the completion of the terms of service for each of these individuals,
+their Owner permissions within the GitHub organization shall be removed.
+
+### Members
+
+GitHub users are added as members to the Node.js GitHub Organization when they
+are added to any Working Group. Organization Owners should add new members
+to the organization when requested by a Working Group.
+
+## Repositories
+
+Any organization member may request the creation of a new repository within the
+Node.js Foundation GitHub Organization. Provided there are no objections from
+any voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be moved to a vote in
+each of the Technical Steering and Community Committees.  A simple majority of
+each group *rejecting* the creation of the repository is required to block
+creating the repository. Such requests must be posted as issues in the
+nodejs/admin repository.
+
+Any repository created under the Node.js GitHub Organization is considered to be
+a project under the ownership of the Node.js Foundation, and thereby subject
+to the Intellectual Property and Governance policies of the Foundation.
+
+No repository may be deleted, transferred in to, or transferred out of, the
+Node.js Foundation GitHub Organization without a simple majority of both
+the TSC and CommComm in *favor* of the action. In certain cases, Node.js
```

Comment:
"In certain cases" - Did you have any such cases in mind?

Reply:
When bringing in a completely new top level project, for instance.